locales.json

```diff
@@ -1690,7 +1690,8 @@
       "noPermission": "Standortzugriff erforderlich",
       "loading": "Bestimme Standort...",
       "notAvailable": "Standortbestimmung deaktiviert",
-      "timeout": "Standortbestimmung fehlgeschlagen"
+      "timeout": "Standortbestimmung fehlgeschlagen",
+      "refresh": "Aktualisiere Orte in der Nähe"
```

Comment:
Maybe "Liste der Orte erneuern/aktualisieren" would be a better fit.

Reply:
Changed to "Liste der Orte aktualisieren"